gammapy/maps/base.py

```diff
@@ -33,11 +34,24 @@ class Map(object):
         Data array
     meta : `~collections.OrderedDict`
         Dictionary to store meta data.
+    unit : str or `~astropy.unit.Unit`
+        The map unit if data is a `~numpy.ndarray`. Default is dimensionless
     """
 
-    def __init__(self, geom, data, meta=None):
+    def __init__(self, geom, data, meta=None, unit=None):
         self._geom = geom
-        self._data = data
+        if isinstance(data,Quantity):
+            self.unit = Quantity.unit.to_string()
```

Comment:
I would suggest to call the data member `self._unit` to be consistent with the rest of the `gammapy.maps` codebase

Reply:
OK. Done.
I added a `Map.unit` property that return a `~astropy.units.Unit`